R/check_target_dataset_unique.R

```diff
@@ -0,0 +1,47 @@
+#' Check that a single unique target dataset exists for a given target type.
+#'
+#' @inheritParams hubData::get_target_path
+#' @inherit check_tbl_col_types return
+#' @export
+check_target_dataset_unique <- function(hub_path,
+                                        target_type = c(
+                                          "time-series",
+                                          "oracle-output"
+                                        )) {
+  target_type <- rlang::arg_match(target_type)
+  file_path <- file.path("target-data", target_type)
+
+  target_path <- hubData::get_target_path(hub_path, target_type)
```

Comment:
I did not get an error if I have both `time-series.csv` and a partitioned version of the file as `target=inc%20death/time-series0.parquet` + `target=inc%20hosp/time-series0.parquet`.

It seems that target_path is an empty character string for the partitioned version.